src/test/java/org/fundacionjala/coding/denis/EanTest.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.coding.denis;
+
+
+import org.junit.Assert;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.junit.Assert'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)